modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -338,9 +340,11 @@ default Subscription subscribe(BiConsumer<? super T, ? super T> changeSubscriber
     }
 
     /**
-     * Creates a {@link Subscription} on this value which immediately provides
-     * the current value to the given {@code valueSubscriber}, followed by any
-     * subsequent values whenever its value changes.
+     * Creates a {@code Subscription} on this {@code ObservableValue} which immediately
+     * provides the current value to the given {@code valueSubscriber}, followed by any
+     * subsequent values whenever its value changes. The {@code valueSubscriber} is called
+     * for convenience since usually the user will want to initialize a value
+     * and then update on changes.
```

Comment:
Only the initial call is for convenience. Subsequent calls are necessary. Do you think it's worth calling this out? Something like:

```
     * ...The {@code valueSubscriber} is called immediately
     * for convenience, since usually the user will want to initialize a value
     * and then update on changes.
```

(I  might also add a comma before since, but that is more a matter of style)